### DISCUSSION TITLE
Why does LTeX use en-US English for my LaTeX document, and not en-ZA English?

### DISCUSSION THREAD

**Discussion by ghost** [valentjn/vscode-ltex#304] [Q&A] [answered]
There is a document that I am working on that LTeX checks using ENG-US and not ENG-ZA as configured. How do I change this please?

**Comment by valentjn** [comment, marked as answer]
Perhaps you're using the babel package and its American English language setting (e.g., with package option `english`, `american`, or `USenglish` &ndash; they all mean en-US). If yes, [LT<sub>E</sub>X changes its language accordingly](https://valentjn.github.io/vscode-ltex/docs/advanced-usage.html#multilingual-latex-documents-with-the-babel-package), and you need to revert the setting via a [magic comment](https://valentjn.github.io/vscode-ltex/docs/advanced-usage.html#magic-comments). If no, it sounds like a bug and [you should open a bug report](https://valentjn.github.io/vscode-ltex/docs/contributing-code-issues.html#how-to-report-bugs).

See also question #245.
